src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -224,6 +260,15 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
     // The _only_ time that's ever appropriate is when the url has changed.
   }, [url]);  // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
+    return autorun(() => {
+      if (phoneRef.current && iframeRef && pluginInfo.textDecorationHandlerInfo) {
```

Comment:
```suggestion
      if (phoneRef.current && pluginInfo.textDecorationHandlerInfo) {
```
By the same argument as above, I don't think the `iframeRef` test is accomplishing anything here and removing it would presumably allow it to be removed from the dependencies below as well.

Given the lack of other dependencies, are we guaranteed that `phoneRef.current` will be set before the `useEffect` runs the first time? Without other dependencies it will never run again.

Reply:
I removed the `iframeRef` dependency, not sure if it needs some other dependency.  My tests cases are all working, but we need to keep an eye on this,